tools/gce-quota-sync/gce-quota-sync.py

```diff
@@ -0,0 +1,177 @@
+#!/usr/bin/env python
+#
+# Copyright 2017 Google Inc.
```

Comment:
Why not 2018?

Reply:
Gah stupid me, copied it from source files I had around... thanks for noticing!